Flow.Launcher.Infrastructure/UserSettings/CustomBrowserViewModel.cs

```diff
@@ -0,0 +1,32 @@
+using Flow.Launcher.Plugin;
+
+namespace Flow.Launcher.Infrastructure.UserSettings
+{
+    public class CustomBrowserViewModel : BaseModel
```

Comment:
why not put this  in the same location as the other vms?

Reply:
because it needs to be accessed in Setting